<launch>
	<node pkg="artiesans" name="face_detection" type="facedetection.py"/>
	<node pkg="artiesans" name="disk_ident" type="disk_ident.py"/>
	<node pkg="artiesans" name="robot_speech" type="robot_speech.py"/>
	<!--<node name="recognizer" pkg="pocketsphinx" type="recognizer.py" output="screen">
		<param name="lm" value="$(find rbx1_speech)/config/5740.lm"/>
		<param name="dict" value="$(find rbx1_speech)/config/5740.dic"/>
	</node>-->
</launch>
